spec/api_spec.rb

```diff
@@ -35,6 +35,14 @@
     end
   end
 
+  it "should allow using derived_next_cursor when listing details of a single resource" do
+    expected = {
+      [:payload, :derived_next_cursor] => "abc"
```

Comment:
Please replace "abc" with some more real value(something that can be a valid cursor), see python SDK for example

Reply:
Python implementation seems to use "foo" (https://github.com/cloudinary/pycloudinary/pull/196/files#diff-62cd34974071de9a77194382ffd0bb70R185) but I've replaced "abc" with a "real" cursor value.